terraform/cloudwatch.tf

```diff
@@ -2,3 +2,31 @@ resource aws_cloudwatch_log_group vphone {
   name              = "vphone"
   retention_in_days = 14
 }
+
+module "notify_slack" {
+  source             = "git@github.com:terraform-aws-modules/terraform-aws-notify-slack.git"
```

Comment:
It looks in git by default - see https://github.com/mvascna/virtual-phone-line/blob/master/terraform/vpc.tf#L2

Reply:
nah dawg

Reply:
it looks in terraform registry by default